docs/manuals/studio/config/elements.md

```diff
@@ -0,0 +1,218 @@
+Configuration elements defined in a configuration file are grouped
+by type; each element type has a specific section in the
+"Taipy Configs" panel.
+
+## Data Nodes configuration
```

Comment:
Not a big deal, but the first-level title should have one # only.

All titles could move to an upper title level.

Reply:
I will chage that. The result with a lower level is better looking.